Common/Data/Language/ENG_MSG.TXT

```diff
@@ -1031,7 +1031,7 @@ _@M975_ " GPS data error "
 #
 # V6.2
 #
-_@M981_ "Auto-Contrast"
+_@M981_ "Auto-Contrast "
```

Comment:
why added space ?

Reply:
because all in this dialog have it:
![image](https://user-images.githubusercontent.com/1188401/51176894-6d699680-18be-11e9-847f-26b98e4704f1.png)